page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/utils/ReflectionUtils.java

```diff
@@ -26,9 +28,36 @@
 import ru.sbtqa.tag.qautils.i18n.I18NRuntimeException;
 import ru.sbtqa.tag.qautils.reflect.FieldUtilsExt;
 
-public class PageFactoryUtils {
+public class ReflectionUtils {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)